meetings/2017-01-04.md

```diff
@@ -0,0 +1,331 @@
+are addressing here?
+
+@joshgav: Seems there is a greater issue of what standard `console` behavior
+should be, and whether we should adhere to existing quasi-standard behaviors.
+
+@jasnell: Comparable to work on URL spec.
+
+@Fishrock123: That’s what the WHATWG console spec is meant to address, but it’s
+not complete.
+
+@MylesBorins: Meta-issue is whether we should work with WHATWG to arrive at
+common conventions. Issue is that browsers will dominate discussion.
+
+@Fishrock123: Much work on timers has been to make Node’s behavior as close to
+browser behavior as possible.
+
+@evanlucas: API needs for console in Node are different than in browsers.
+
+@bmeck: Inspector hooks into our Console API. So console.log output is captured
+by the inspector as well. It patches console.log to do this.
+
+@jasnell: Likely because we’ve never given an API to hook into console output.
+
+@bmeck: They also add console.group and others.
+
+@Fishrock123: Question is: Are we interested in following a proper console spec?
+
+@jasnell: Should this consideration hold up this issue/PR?
+
+@evanlucas: This issue could be handled in userland, like the ‘printf’ module.
+
+@bmeck: We actually ship two different consoles - the Inspector console API is
+different.
+
+Differences between our console and WHATWG console are mostly documented in this
+issue and its inline links.
+
+**Next steps**:
+
+- More investigation into WHATWG work, if we should participate in that work,
+  and if we should follow the final spec.
+- Continue discussion in GitHub.
+
+
+---
+
+### timers: cleanup extraneous property on Immediates [#10205](https://github.com/nodejs/node/pull/10205)
+
+@Fishrock123: An extra property (\_callback) was added 2-3 months ago which just
+copies another property (\_onImmediate) but actually behaves a bit differently
+2-3 months ago. Other similar extraneous properties from similar APIs have been
+removed; this seems to be of the same category.
+
+@jasnell: Any evidence that it’s actually in use?
+
+@trott: Okay with landing this PR, but concerned about hand-wavey nature of
+determining whether something is semver-major.
+
+@jasnell: If it’s in 6 we should take it through a deprecation cycle, if it’s
+only in 7 we can remove immediately.
+
+@mhdawson: If it’s in an LTS it should go through a deprecation cycle.
+
+@jasnell: For clarity to ecosystem it’s better to take these through a
+deprecation cycle.
+
+@Fishrock123: Adding deprecation wrapper may slow down/deopt setImmediate.
+
+@jasnell: If we ensure our code avoids the deprecated property we would avoid
+any deopt or slow down.
+
+@trevnorris: Almost everything in Node is somehow accessible publicly. Does this
+mean that any internal change has to be deprecated first?
+
+@jasnell: We need a better policy to define what must be deprecated.
+
+And perhaps to define what is internal and what is a public API.
+
+@trott: Should we hold on landing this item for the larger discussion?
```

Comment:
I think what I was saying was that we probably shouldn't hold on landing this. Since this probably doesn't add anything to the conversation maybe it can be removed?